### PR TITLE
Skip unittest on certain pytorch version due to API behavior change

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
Fixes # 1149.

### Description
Run unittest with expected results on certain pytorch version only

### Status
**Work in progress**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by Nic-Ma** [comment]
Hi @IsaacYangSLA ,

@wyli and I already covered your fix in the previous PR: https://github.com/Project-MONAI/MONAI/pull/1162
Currenly, the only remaining issue is the shuffle logic in DataLoader changed in PyTorch 1.7.0, we need to figure out a solution to get the `generator` and set to DataLoader, then the test results for PyTorch 1.7.0 & 1.5.0 will be same.
Can you work on that PR directly? What do you think?

Thanks.

**Comment by wyli** [comment]
closing this in favour of #1162
